tests/unit/lib/SGMatrix_unittest.cc

```diff
@@ -13,6 +13,22 @@
 
 using namespace shogun;
 
+TEST(SGMatrixTest, vectorized_rng)
+{
+	auto rng_check_float = [](auto elem) {
+		SGMatrix<decltype(elem)> sg_mat(10, 5);
```

Comment:
would there be a away to check it against the individual calling of random with some seed fixing magic?
If that's a pain, dont bother, this test is more to check that no memory errors happen I guess

Reply:
might be a bit of a chore to implement - especially because I'd have to based on the type use the `random_half_open()` vs. `fill_array_oc` version.

Reply:
yeah don't worry then